.github/dependabot.yml

```diff
@@ -7,13 +7,21 @@ updates:
       interval: "weekly"
     commit-message:
       prefix: "chore"
+    groups:
+      all:
+        patterns:
+        - "*"
```

Comment:
might want to group specific things here too, like docuausuarus ones instead of *